hw/ip/lc_ctrl/data/lc_ctrl.hjson

```diff
@@ -25,7 +25,7 @@
     {clock: "clk_kmac_i", reset: "rst_kmac_ni"}
   ]
   bus_interfaces: [
-    { protocol: "tlul", direction: "device" }
+    { protocol: "tlul", direction: "device", name: "regs" }
```

Comment:
Crikey, this commit was big! If the rest of this PR gets held up at all, it might be worth splitting out this commit and getting it merged. I think it looks right, but it was long and I'd rather not read through it again :-)